admin_app/package.json

```diff
@@ -17,14 +17,16 @@
     "@mui/material": "^5.16.5",
     "@types/google.accounts": "^0.0.14",
     "@types/papaparse": "^5.3.14",
+    "@types/styled-components": "^5.1.34",
```

Comment:
I suggest we stick to MUI as much as possible. You Can use a Box and pass styles to "sx = {{ ... }}" to style it.